src/components/users/change-password/ko/runtime/change-password.ts

```diff
@@ -140,38 +131,8 @@ export class ChangePassword {
                 await this.usersService.changePassword(userId, this.newPassword());
             }
             this.isChangeConfirmed(true);
+        }, this.eventManager, errorSources.changepassword, undefined, detail => `${detail.target}: ${detail.message} \n`);
 
-            const validationReport: ValidationReport = {
-                source: "changepassword",
-                errors: []
-            };
-            this.eventManager.dispatchEvent("onValidationErrors", validationReport);
-        } catch (error) {
-            if (isCaptcha) {
-                await this.refreshCaptcha();
-            }
```

Comment:
this block is required for refresh the captcha widget on error.

Reply:
I totally missed that, my bad, fixed. Great catch!